CHANGELOG.md

```diff
@@ -12,20 +12,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Dependencies
 - Bump `log4j-core` from 2.18.0 to 2.19.0
+- Bumps `reactor-netty-http` from 1.0.24 to 1.1.2
```

Comment:
The changelog is a bit messed up :(

Reply:
Ooops must have messed up with the rebase on my end.